### PR TITLE
#165305766 Fix listing of users 

### PR DESCRIPTION
**Pull request by mdn-network**
#### Description
- Fix response to exclude user id fields 
#### Type of change
- [x] Bug fix (nonbreaking change which fixes an issue)
- [ ] New feature (nonbreaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### How Has This Been Tested?
Please describe the tests that you ran to verify your changes
- [ ] End to end
- [ ] Integration

#### Checklist:
- [x] My code follows the style guide for this project
- [x] I have linted my code prior to submission
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[165305766](https://www.pivotaltracker.com/story/show/165305766)

### CODE REVIEW

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]
Hello @Philipotieno ,
You can squash the commits so that we have one commit per PR as we agreed

**Comment by mdn-network** [comment]
@Ogutu-Brian I have already
Thanks for the feedback

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by matthenge — APPROVED** [review]
LGTM

**Review by IssaIan — DISMISSED** [review]

File: authors/apps/authentication/serializers.py [hunk 1]
```diff
@@ -44,25 +50,26 @@ class RegistrationSerializer(serializers.ModelSerializer):
...
         validators=[
             UniqueValidator(
                 queryset=User.objects.all(),
-                message='username already taken',
+                message='user with this username already exists',
             )
         ],
         error_messages={
-            'invalid': 'Username should have no spaces or special characters only',
+            'invalid': 'Username should be at least 3 characters, have no spaces or special characters except hyphen-, fullstop. and underscore_',
```
Comment (oluwagbenga-joloko) [inline authors/apps/authentication/serializers.py:72]: Why were all these changes added ? should this not be for a different task to fix signup validation error messages. @Philipotieno

**Pull request closed without merging**